src/mesh/index_derivs.cxx

```diff
@@ -1407,9 +1409,7 @@ const Field3D Mesh::indexDDZ(const Field3D &f, CELL_LOC outloc, DIFF_METHOD meth
 
 const Field2D Mesh::indexDDZ(const Field2D &f) {
   ASSERT1(this == f.getMesh());
```

Comment:
Do we need/want similar inserts in the other routines that are changed here or is this already handled?

Reply:
Just checked, this is done for all the routines here, just not consistently the first thing.